master/buildbot/schedulers/trysched.py

```diff
@@ -117,6 +120,9 @@ def activate(self):
 
     @defer.inlineCallbacks
     def deactivate(self):
+        if not self.enabled:
```

Comment:
We have a CI so that your unit tests are run against all version of twisted/python.

So you just have to write the unit tests, and then they will run everywhere when you push your branch